openpype/lib/log.py

```diff
@@ -493,3 +494,14 @@ def timed(*args, **kw):
             print('%r  %2.2f ms' % (method.__name__, (te - ts) * 1000))
         return result
     return timed
+
+
+def debug_log_instance(logger, msg, instance):
+    """Helper function to write instance.data as json"""
+    def _default_json(value):
+        return str(value)
+
+    logger.debug(msg)
+    logger.debug(
+        json.dumps(instance.data, indent=4, default=_default_json)
+    )
```

Comment:
This is lib (and very specific) function in loggers logic. And I think it should not log but just convert the passed data into json format. Move it to `openpype/lib/config.py` (there are uncategorized lib functions now) and remove logging logic.

Example:
```
def _default_json_convert(value):
    return str(value)


def debug_data_dump(data, indent=4, convert_func=None):
    if convert_func is None:
        convert_func = _default_json_convert
    return json.dumps(data, indent=indent, default=convert_func)
```